From 3b029441f506dd85c6d4c827f2616d0076b7b8e9 Mon Sep 17 00:00:00 2001
From: Ovidiu Sas <osas@voipembedded.com>
Date: Thu, 26 Mar 2015 21:42:57 -0400
Subject: [PATCH] registrar: fix compile errors introduced by commit
 b86ff789b86d2f7512c23fe0624d553b37ce11cb  - define w_registered3  - define
 w_registered4  - define registered_fixup

---
 modules/registrar/reg_mod.c | 3 +++
 1 file changed, 3 insertions(+)

--- a/modules/registrar/reg_mod.c
+++ b/modules/registrar/reg_mod.c
@@ -95,6 +95,8 @@
 static int w_registered(struct sip_msg* _m, char* _d, char* _uri);
 static int w_unregister(struct sip_msg* _m, char* _d, char* _uri);
 static int w_unregister2(struct sip_msg* _m, char* _d, char* _uri, char *_ruid);
+static int w_registered3(struct sip_msg* _m, char* _d, char* _uri, char* _flags);
+static int w_registered4(struct sip_msg* _m, char* _d, char* _uri, char* _flags, char* _actionflags);
 
 /*! \brief Fixup functions */
 static int domain_fixup(void** param, int param_no);
@@ -102,6 +104,7 @@
 static int save_fixup(void** param, int param_no);
 static int unreg_fixup(void** param, int param_no);
 static int fetchc_fixup(void** param, int param_no);
+static int registered_fixup(void** param, int param_no);
 /*! \brief Functions */
 static int add_sock_hdr(struct sip_msg* msg, char *str, char *foo);
 
